models/fv_rs_cons_if.go

```diff
@@ -0,0 +1,99 @@
+package models
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/ciscoecosystem/aci-go-client/container"
+)
+
+const (
+	DnfvRsConsIf        = "uni/tn-%s/ap-%s/epg-%s/rsconsIf-%s"
+	RnfvRsConsIf        = "rsconsIf-%s"
+	ParentDnfvRsConsIf  = "uni/tn-%s/ap-%s/epg-%s"
+	FvrsconsifClassName = "fvRsConsIf"
+)
+
+type ContractInterface struct {
+	BaseAttributes
+	NameAliasAttribute
+	ContractInterfaceAttributes
+}
+
+type ContractInterfaceAttributes struct {
+	Annotation   string `json:",omitempty"`
+	Prio         string `json:",omitempty"`
+	TnVzCPIfName string `json:",omitempty"`
+}
+
+func NewContractInterface(fvRsConsIfRn, parentDn, description, nameAlias string, fvRsConsIfAttr ContractInterfaceAttributes) *ContractInterface {
+	dn := fmt.Sprintf("%s/%s", parentDn, fvRsConsIfRn)
+	return &ContractInterface{
+		BaseAttributes: BaseAttributes{
+			DistinguishedName: dn,
+			Description:       description,
```

Comment:
Not sure if we can remove it here but description is not supported on this object